ProjectLighthouse.Servers.Website/Pages/UserPage.cshtml.cs

```diff
@@ -29,35 +31,25 @@ public class UserPage : BaseLayout
     public List<SlotEntity>? QueuedSlots;
 
     public UserEntity? ProfileUser;
+
+    public bool CanAccessProfile;
+    public bool CanAccessSlots;
```

Comment:
This might infer context better if it were named CanViewProfile and CanViewSlots